BinaryEncoding.md

```diff
@@ -142,33 +133,38 @@ A module may contain at most one import table section.
 #### Import entry
```

Comment:
My plan was, after this and the other two PRs merge, I'd make a new PR that specified the precise order of all sections (as a simple regexp: `signatures? import_table? function_signatures? ...`) and reordered the sections in BinaryEncoding.md.  This would conflict with changes in this PR so I was holding off until after they merged.